gradle.properties

```diff
@@ -19,3 +19,4 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+APPID = "16ed760ab156214729d1e276a01e9653";
```

Comment:
You have to add this APPID on the gradle

Reply:
Done thx